### DISCUSSION TITLE
make a new plot with 1.1.7 version

### DISCUSSION THREAD

**Discussion by Jimmy01240397** [Chia-Network/chia-blockchain#7439] [Support]
I accidentally used an old version to create plots. Are these plots valid?
![image](https://user-images.githubusercontent.com/57281249/125411696-bd266b80-e3f0-11eb-80e6-f97f29476b8d.png)
![image](https://user-images.githubusercontent.com/57281249/125411747-8a2a5673-1750-4c61-b9f3-adf94f0f6866.png)
ps:(I used chia plots check with 1.2.0 is pass)

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
